.github/workflows/test-linux-cpu.yml

```diff
@@ -55,3 +55,6 @@ jobs:
         # Run Tests
         python3 -m torch.utils.collect_env
         python3 -m pytest --junitxml=test-results/junit.xml -v --durations 20
+
+        # Specific test for warnings on "from torchvision.datasets import wrap_dataset_for_transforms_v2"
```

Comment:
```suggestion
        # Specific test for warnings on "from torchvision.datasets import wrap_dataset_for_transforms_v2"
        # We keep them separate to avoid any side effects due to warnings / imports.
        # TODO: Remove this and add proper tests (possibly using a sub-process solution as described
        # in https://github.com/pytorch/vision/pull/7269).
```